aio/content/marketing/docs.md

```diff
@@ -3,8 +3,8 @@
 Angular es un framework de diseño de aplicaciones y plataforma de desarrollo para crear aplicaciones de una
 sola página eficientes y sofisticadas
 
-Esta documentación de Angular te ayuda a aprender y usar el framework y la plataforma de desarrollo, desde su
-primera aplicación hasta la optimización de aplicaciones complejas de una sola página, para empresas.
```

Comment:
dice: "for entreprises", aunque si fuese "enterprises applications" aplica la sugerencia qie indicas.
![image](https://user-images.githubusercontent.com/29555498/94098640-b2adcd80-fdee-11ea-8134-eeb64842e2b6.png)

Reply:
Listo, he dejado la oración en la forma original. @christianmorante